articles/tokens/id_token.md

```diff
@@ -1,130 +1,84 @@
     params: {scope: 'openid name email'}
   }
-}; 
+};
 
 var lock = new Auth0Lock(
-  'YOUR_CLIENT_ID',
-  'YOUR_NAMESPACE',
+  ${account.clientId},
+  ${account.namespace},
   options
 );
 
 lock.show();
 ```
 
-The above sample, specifying `openid name email` will result in a JWT with claims for the name and email attributes within the user profile.  The `responseType` should be `token` for client-side authentication flows and `code` for server-side authentication flows as described for the `/authorize` endpoint in the [authentication API](https://auth0.com/docs/api/authentication)
-
-The scope of the id_token JWT can also be altered via Rules, through the context.jwtConfiguration.scopes object as documented [here](/rules/context)
-
-There is a [sample for altering scopes in a Rule](https://github.com/auth0/rules/blob/dff2a3e72f01d33af3086414be7cf115b19eea0c/rules/custom-scopes.md)
-
-Additional information on the `id_token` is [here](/jwt)
-
-A valid JWT can be pasted into the [jwt.io website](https://jwt.io) to view the contents of the JWT.
-
-A [blog entry on JWT](https://auth0.com/blog/2015/07/21/jwt-json-webtoken-logo/) provides an explanation of why it is getting to be more popular.
-
-[Additional samples](https://github.com/auth0/auth0.js) show use of the auth0.js library.
+Again, the `responseType` must be set to `id_token` in order to get one back. The `id_token` will contain only the claims specified as the value of the `scope` parameter (in this example, `openid name email`).
 
 ## Validity
 
-The `id_token` is valid for 10 hours (36000 seconds) by default.  The expiration of this token can be set in the `Apps/APIs` -> Settings screen using the `JWT expiration` field.
-
-The validity period of the token can be altered via Auth0 Rules using the context.jwtConfiguration.lifetimeInSeconds object as documented [here](/rules#context)
-
-For example, the following code in a Rule would set the id_token JWT expiration to 1 hour.
-```
-context.jwtConfiguration.lifetimeInSeconds = 3600;
-```
+The `id_token` is valid for 10 hours (36000 seconds) by default.  The value can be changed in the [Dashboard > Clients > Settings](${manage_url}/#/clients/${account.clientId}/settings) screen using the `JWT Expiration (seconds)` field.
 
-## Renewing the token
+## Renewal of the token
 
-A new `id_token` can be obtained using an existing, unexpired `id_token` or by using a refresh token and the `/delegation` endpoint.
+There are cases where you might want to renew your `id_token`. The most common case is that you want to refresh the claims contained in the `id_token` and see that they are still valid. For example, the `id_token` may contain the user's authorization claims and you want to check that their rights are valid before allowing sensitive operations.
 
-To use an existing, unexpired `id_token` to obtain a new one, use the `renewIdToken` function within auth0.js library as shown [here](https://github.com/auth0/auth0.js#refresh-token)
```

Comment:
I removed this because the `renewIdToken` function calls the `getDelegationToken` method, which calls the `/delegation` endpoint https://github.com/auth0/auth0.js/blob/3e625778ca00a843129f28ca2a71fd9d8beb96b2/index.js#L1563